content/blog/peace-of-mind-with-cloud-secret-providers/index.md

```diff
@@ -0,0 +1,591 @@
+---
+title: "Peace of Mind with Cloud Secret Providers"
+
+date: 2020-04-29T14:26:35-07:00
+draft: true
+meta_desc: "Learn how to encrypt your Pulumi secrets with a cloud secrets provider to ensure secrets never leave your control"
+meta_image: meta.png
+authors:
+    - lee-briggs
+tags:
+    - features
+    - security
+---
+
+The secrets in your infrastructure are a vital part of your security model, and provisioning infrastructure is an inherently privileged process. Back in [Pulumi 0.17.12](https://www.pulumi.com/blog/managing-secrets-with-pulumi/) we introduced client-side secret encryption and started encrypting secret configuration values inside the Pulumi state so that users could be confident their passwords, tokens and other secret values were viewable only by them while managing their infrastructure.
```

Comment:
```suggestion
The secrets in your infrastructure are a vital part of your security model, and provisioning infrastructure is an inherently privileged process. Back in [Pulumi 0.17.12](https://www.pulumi.com/blog/managing-secrets-with-pulumi/), we introduced client-side secret encryption and encrypted secret configuration values inside the Pulumi state so that users could be confident their passwords, tokens, and other secret values were viewable only by them while managing their infrastructure.
```